utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -571,6 +572,9 @@ def create_argument_parser():
     option(['--skstresstester'], store_true('build_skstresstester'),
            help='build the SourceKit stress tester')
 
+    option(['--swiftformat '], store_true('build_swiftformat'),
+           help='build the SourceKit stress tester')
```

Comment:
The help comments here and below need to be updated.

Reply:
Oops. Thanks!